### DISCUSSION TITLE
Which two countries have the longest land border in the world?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#588] [Galaxy Trivia] [answered]
**Bonus:** Where is the second longest border?

**Comment by alejandroch1202** [comment]
- Canada's and United States
- Russia and Kazakhstan

**Comment by peterbaccaro** [comment, marked as answer]
The border between the **United States** and **Canada** is approximately 8,891 kilometres (5,525 miles) long, making it the longest international land border in the world. 

The second longest border is the border between **Russia** and **Kazakhstan** which is approximately 7,511 kilometres (4,649 miles) long.
